src/main/java/org/fundacionjala/coding/jimmy/CesarCrypto.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.jimmy;
+
+
+/**
+ * Class for developing the methods.
+ */
+public class CesarCrypto {
```

Comment:
It is missing the vigenere cipher 
also you should implement the enconde and decode.